### DISCUSSION TITLE
Deploying .Net Core 6 Web API Project

### DISCUSSION THREAD

**Discussion by prashantchothani** [vercel/community#2827] [Help] [answered]
### Summary

How can we deploy a .Net Core 6 Web API Project on Vercel ?

### Steps to Reproduce

Cannot find any documentation on how to do that.

**Comment by amyegan** [comment, marked as answer]
Hi @prashantchothani. We don't currently have a .NET Core [Serverless Function Runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#) available. If you're interested, you may be able to create your own [Community Runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#community-runtimes) to add support for .NET on Vercel.

Another approach is hosting the .NET Core API with a service tailored to .NET backend and then deploying only a frontend to Vercel

**Reply by TheHolyRoller** [reply]
Hi I'm just wondering how could I go about deploying the front end on Vercel? Could I use .Net or would I have to use a supported js framework like Nuxt or Next?

**Reply by amyegan** [reply]
Yes, you would need to use a supported frontend framework

**Reply by TheHolyRoller** [reply]
Okay thanks clarifying
